dev-data/about-course.data.tsx

```diff
@@ -35,6 +36,26 @@ const preSchoolIntro = {
     'Подготовительный этап поможет тем, кто мало знаком или совсем не знаком с программированием и хотел бы впоследствии учиться на основном курсе «JavaScript/Front-end».',
 };
 
+// const infoMaterialsTextEn = ['All materials are publicly available on the', 'channel and'];
+// const infoMaterialsTextRu = ['Все материалы находятся в открытом доступе на канале', 'и'];
+
+const infoMaterialsEn = (
+  <Paragraph>
+    All materials are publicly available on the
+    {' '}
+    <LinkCustom href="https://www.youtube.com/c/RSschool" external>
+      YouTube
+    </LinkCustom>
+    {' '}
+    channel and
+    {' '}
+    <LinkCustom href="https://docs.rs.school" external>
```

Comment:
maybe we can use DOCUMENTATION_LINKS here?
```suggestion
    <LinkCustom href={DOCUMENTATION_LINKS.EN} external>
```